src/main/webapp/style.css

```diff
@@ -118,6 +118,34 @@ input[type=submit]:hover {
   transition: all .4s ease-in-out;
 }
 
+#year-slider {
+  width: 100%;
+}
+
+#set-year {
+  -moz-appearance: none;
+  -webkit-appearance: none;  /* Override default style for Chrome and Mozilla */
+  background: #d3d3d3;
+  border-radius: 25px;
+  height: 10px;
+  outline: none;
+  width: 40%;
+}
+
+/* The handle for the set-year slider (once for Chrome, once for Mozilla) */
+#set-year::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  background: #0dbd00;
```

Comment:
not sure if this is just me, but when I changed the color of the map the green button sometimes clashed with the different color scheme. I was thinking it might be cool to change the slider to match the color picker, but I'm curious to hear other thoughts.